lib/dor/assembly/content_metadata/file_set.rb

```diff
@@ -10,14 +10,14 @@ class FileSet
         # if input file has one of these extensions in a 3D object, it will get the 3d resource type
         VALID_THREE_DIMENSION_EXTENTIONS = ['.obj'].freeze
 
-        # @param style
+        # @param [String] object_type
         # @param [Array<Assembly::ObjectFile>] resource_files
-        def initialize(resource_files:, style:)
+        def initialize(resource_files:, object_type:)
           @resource_files = resource_files
-          @style = style
+          @object_type = object_type
         end
 
-        # otherwise look at the style to determine the resource_type_description
+        # otherwise look at the object_type to determine the resource_type_description
```

Comment:
"otherwise" is an odd way to start a method comment  - can you please improve on it?

Reply:
I'm just going to delete this comment. I'm not sure what it's trying to say.